2024/fall/Scheffler-MW@11am/alen-rubilar.md

```diff
@@ -0,0 +1,3 @@
+# Hello, there!
+
+My name is **Alen Rubilar**, I'm an M26 from Chile double majoring in ~~I don't really know what I'm doing~~ Computer Science and Natural Science
```

Comment:
uhm akshually?

Reply:
cool beans 😎

Reply:
idk what ure doing is whimsical enough eh? then again I don't doubt ur md skills so APPROVED
![image](https://github.com/user-attachments/assets/5267173f-3181-4199-87c6-f295d160f119)